addons/website_slides/views/mail_templates.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_template_data_new_rank_reached" model="mail.template">
```

Comment:
Side note: commit mismatch for the template in slides (currently included in 
[IMP] gamification : add ranks based on karma whether it should probably be moved to gamification, and then modules will override the famous button thingy).